README.md

```diff
@@ -13,12 +13,45 @@
 
 * [Description](#description)
 * [License](#license)
+* [Usage](#usage)
+  * [Tuning Configuration](#tuning-configuration)
 * [Authors](#authors)
 
 ## Description
 
 This module configures the [Grafana Alloy](https://grafana.com/docs/alloy/latest/) service.
 
+## Usage
+To use that module, you need to call the main class either via puppet :
+```puppet
+include grafana_alloy
+```
+either
+```puppet
+---
+classes:
+  - grafana_alloy
+```
+
+### Tuning configuration
+If you want to adapt the Alloy configuration you can use the `config` variable like this (into hiera declaration file):
+
+```puppet
+grafana_alloy::config:
```

Comment:
`grafana_alloy::config` is a private API so documenting it as the primary usage doesn't look like the best idea. Perhaps better to expose it in `init.pp` as a real parameter?